shell/browser/native_browser_view_views.h

```diff
@@ -40,6 +48,8 @@ class NativeBrowserViewViews : public NativeBrowserView {
   float auto_vertical_proportion_height_ = 0.;
   float auto_vertical_proportion_top_ = 0.;
 
```

Comment:
where can I find `-webkit-app-region` parsing?
I wanna find out how Electron works if `-webkit-app-region` is applied.
@ckerr @codebytere @lsegal @vitaliel @anaisbetts